crates/client/sync/src/eth/src/config.rs

```diff
@@ -0,0 +1,143 @@
+//! Base Ethereum client configuration.
+//!
+//! Use it as is or reuse top-level fields in your config ("inherit")
+//! in order to share the same configuration file.
+//!
+//! struct MyServiceConfig {
+//!     pub provider: EthereumProviderConfig,
+//!     pub wallet: Option<EthereumWalletConfig>,
+//!     pub contracts: Option<StarknetContracts>,
+//! }
+//!
+//! Default provider and wallet configurations are set for use with Anvil
+//! - local Ethereum environment.
```

Comment:
I didn't really get this comment. I think it's from Madara. Do you know what this means or do you want to change it?

Reply:
yes not required, removed